include/dwg.h

```diff
@@ -8075,6 +8075,308 @@ enum {
   OPTS_R11_XDATA = 128, //?
 };
 
+enum {
+  OPTS_R11__3DFACE_HAS_Z_FIRST = 1,
+  OPTS_R11__3DFACE_HAS_Z_SECOND = 2,
+  OPTS_R11__3DFACE_HAS_Z_THIRD = 4,
+  OPTS_R11__3DFACE_HAS_Z_FOURTH = 8,
+  OPTS_R11__3DFACE_UNKNOWN_16 = 16,
```

Comment:
I prefer using the R11 types instead, esp. for DIMENSION, POLYLINE and VERTEX later.
And I prefer to skip all the unused flags.

Reply:
DIMENSION, POLYLINE, VERTEX are commons entities, we need somehow unify.

btw: These unused flags are good to check existing flags in real DWG files.

Reply:
I see the reason why, but I rather keep only used flags, and maybe warn when some unused flag appears out of the blue.

Reply:
If you have a way how to print that flag that is not defined, that's ok for me.

Reply:
See 23b1fc8c5d245c73bca299cf958d983e0208e93f
cleanup unused r11 opts and flags